CHANGELOG.md

```diff
@@ -1,8 +1,26 @@
-# Release Notes for Kubernetes 1.2.0
+<!--THIS TOC NEEDS TO BE GENERATED EVERY TIME THIS FILE IS UPDATED -->
```

Comment:
For an example of how to autogenerate a TOC, see https://raw.githubusercontent.com/kubernetes/kubernetes/master/docs/devel/api_changes.md